lib/components/viewers/stop-viewer.js

```diff
@@ -233,48 +202,64 @@ class StopViewer extends Component {
           type='date'
           value={this.state.date}
           style={{
-            width: '115px',
+            width: '125px',
             border: 'none',
             outline: 'none'
           }}
           required
           onChange={this.handleDateChange}
         />}
+
+        {timezoneWarning}
       </div>
     )
   }
 
   render () {
     const {
+      autoRefreshStopTimes,
+      findStopTimesForStop,
       homeTimezone,
       stopData,
       stopViewerArriving,
       stopViewerConfig,
       timeFormat,
-      transitOperators
+      toggleAutoRefresh,
+      transitOperators,
+      viewedStop
     } = this.props
-    const { scheduleView, spin } = this.state
+    const { date, scheduleView } = this.state
     const hasStopTimesAndRoutes = !!(stopData && stopData.stopTimes && stopData.stopTimes.length > 0 && stopData.routes)
-    // construct a lookup table mapping pattern (e.g. 'ROUTE_ID-HEADSIGN') to
-    // an array of stoptimes
-    const stopTimesByPattern = getStopTimesByPattern(stopData)
-    const routeComparator = coreUtils.route.makeRouteComparator(
-      transitOperators
-    )
-    const patternHeadsignComparator = coreUtils.route.makeStringValueComparator(
-      pattern => pattern.pattern.headsign
-    )
-    const patternComparator = (patternA, patternB) => {
-      // first sort by routes
-      const routeCompareValue = routeComparator(
-        patternA.route,
-        patternB.route
-      )
-      if (routeCompareValue !== 0) return routeCompareValue
 
-      // if same route, sort by headsign
-      return patternHeadsignComparator(patternA, patternB)
+    let contents
+    if (!hasStopTimesAndRoutes) {
+      contents = <div>No stop times found for date.</div>
```

Comment:
"No stop times" flashes now when you click the refresh button.

Reply:
It was introduced by [this](https://github.com/opentripplanner/otp-react-redux/pull/292/files#diff-341585469c20207f4fd902aa6736bc338f82783a760ce424a8877c305984a5a1R577) which is what we want for the schedule... but not the live arrivals then.

Reply:
Fixed in https://github.com/opentripplanner/otp-react-redux/pull/292/commits/08253eff2bb186584000d7209a5635d6de86dbfb.